Sources/Subs.php

```diff
@@ -1744,6 +1750,12 @@ function parse_bbc($message, $smileys = true, $cache_id = '', $parse_tags = arra
 				'disallow_children' => array('email', 'ftp', 'url', 'iurl'),
 				'disabled_after' => ' ($1)',
 			),
+			array(
+				'tag' => 'youtube',
+				'type' => 'unparsed_content',
+				'content' => '<div class="videocontainer"><div><iframe frameborder="0" src="https://www.youtube.com/embed/$1?wmode=opaque" data-youtube-id="$1" allowfullscreen=""></iframe></div></div>',
+				'disabled_content' => '<a href="https://www.youtube.com/watch?v=$1" target="_blank" rel="noopener">https://www.youtube.com/watch?v=$1</a>',
```

Comment:
This can just be `allowfullscreen` instead of `allowfullscreen=""` since it is a boolean attribute and the presence implies True.